projects/demos/app/demo-modules/kitchen-sink/template.html

```diff
@@ -108,7 +108,7 @@ <h3>
       <tr>
         <th>Title</th>
         <th>Primary color</th>
-        <th>Secondary color</th>
+        <th>Secondary color / foreground</th>
```

Comment:
Actually, let's call this `Secondary + text color`

Reply:
Yeah sure, just go ahead!